MSSQL/OMOPLoader.sql

```diff
@@ -59,15 +59,15 @@ GO
 -- You will almost certainly need to edit your database name
 -- Synonyms for dimension tables
 -- NOTE If this synonym doesn't work, run pcornet_prep and try again
-create synonym i2b2visit for i2b2demodata..visit_dimension

```

Comment:
Same thing with these, project specific stuff can be in your local copy or in a sample preamble for Partners, but shouldn't be in the main transform.

Reply:
Agreed. Reverted in latest commit.